vignettes/reprex-dos-and-donts.Rmd

```diff
@@ -80,7 +80,7 @@ The reprex code:
 
   * Should be easy for others to digest, so **they don't necessarily have to run it**. You are encouraged to include selected bits of output. :scream:
 
-  * Should be easy for others to copy + paste + run, **if they so choose**. Don't let inclusion of output break executability.
+  * Should be easy for others to copy + paste + run, **iff(if and only if) they so choose**. Don't let inclusion of output break executability.
```

Comment:
Add a space between iff and parens?

Reply:
Let's just replace "iff" with "if and only if".